### PR TITLE
Camembert

### PR DESCRIPTION
**Pull request by lionelB**
- camenbert FTW

### CODE REVIEW

**Comment by revolunet** [comment]
awesome :) on a une idée de l'impact ?

File: packages/code-du-travail-nlp/requirements.txt [hunk 1]
```diff
@@ -1,9 +1,7 @@
...
-gunicorn==20.0.0
-sentencepiece
-tf-sentencepiece==0.1.82
-tensorflow_hub
-tensorflow==1.13.1
+gunicorn==19.9.0
 gevent
-unidecode
+git+https://github.com/pytorch/fairseq.git
+torch==1.2.0
```
Comment (lionelB) [inline packages/code-du-travail-nlp/requirements.txt:6]: why pin version 1.2 and use a pytorch 1.3 image ?

**Comment by lionelB** [comment]
it's working \o/

**Pull request closed without merging**